server/game/cards/13-O4B/CarterRichardson.js

```diff
@@ -0,0 +1,61 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class CarterRichardson extends DudeCard {
+    constructor(owner, cardData) {
+        super(owner, cardData, true);
+    }
+    setupCardAbilities(ability) {
+        this.playAction({
+            title: 'Carter Richardson: shootout',
+            playType: 'shootout',
+            cost: ability.costs.payReduceableGRCost(),
+            condition: () => this.game.shootout && this.game.shootout.getPosseByPlayer(this.controller.getOpponent()).getDudes(dude => dude.isWanted()).length,
```

Comment:
I would rather have also check if posse is defined/not null. It may be possible for the shootout to start, but for the opposing posse to yet be formed.

Reply:
ok, will throw that in there.

Reply:
fixed :heavy_check_mark: